services/worker-manager/test/fakes/schemas/azure-vm.yml

```diff
@@ -0,0 +1,74 @@
+# This is a partial schema for Azure VM parameters.  Feel free to add additional
+# properties here if necessary, consulting with the relevant bits of Azure documentation.
+#
+# NOTE: the `testProperty` properties here are used in the tests to verify that
+# additional configuration from the user is correctly passed to the Azure endpoint.
+#
+# https://docs.microsoft.com/en-us/javascript/api/@azure/arm-compute/virtualmachine?view=azure-node-latest
+type: object
+properties:
+  subnetId: {type: string}
+  location: {type: string}
+  tags:
+    type: object
+    additionalProperties: {type: string}
+  # https://docs.microsoft.com/en-us/javascript/api/@azure/arm-compute/windowsconfiguration?view=azure-node-latest
+  osProfile:
+    type: object
+    properties:
+      computerName: {type: string}
+      adminUsername: {type: string}
+      adminPassword: {type: string}
+      customData:
+        type: string
+        pattern: '^[a-zA-Z0-9+/]+=*$' # base64
+      testProperty: {}
+    additionalProperties: false
+    required: [computerName, adminUsername, adminPassword, customData]
+  # https://docs.microsoft.com/en-us/javascript/api/@azure/arm-compute/storageprofile?view=azure-node-latest
+  storageProfile:
+    type: object
+    properties:
+      osDisk:
```

Comment:
We'll add datadisks here for the multi disk deletion PR.

Reply:
Yep, and we can add more stuff as we add features for it.